src/rashdf/plan.py

```diff
@@ -1279,6 +1383,16 @@ def get_meteorology_precip_attrs(self) -> Dict:
         """
         return self.get_attrs(self.PRECIP_PATH)
 
+    def get_obs_data_attrs(self) -> Dict:
```

Comment:
Seems like there aren't any attrs on the `Event Conditions/Observed Data` group? Might want to remove this method.

Reply:
Method removed